skrf/mathFunctions.py

```diff
@@ -267,7 +267,7 @@ def magnitude_2_db(z: NumberLike, zero_nan: bool = True):
     """
     out = 20 * npy.log10(z)
     if zero_nan:
-        return npy.nan_to_num(out, nan=LOG_OF_NEG, neginf=-npy.inf)
+        return npy.nan_to_num(out, copy=False, nan=LOG_OF_NEG, neginf=-npy.inf)
```

Comment:
You need to remove `copy=False` for numpy 2 compatibility :)

Reply:
Yeah, I failed my merging...

Reply:
OK now. The on hp8510c.py comes from pre-commit/ruff